f600233d1b154d507b782f84604bcb12/update/sys_script_include_b29002b097a211106fa8b4b3f153afc4.xml

```diff
@@ -703,14 +684,13 @@ Igce.prototype = {
                 fundingDocument = {
                     "fundingType": fundingRequest.funding_request_type.toString(),
                     "miprNumber": fundingRequest.mipr.mipr_number.toString()
-                }
-            }
-            else if (fundingRequest.funding_request_type == "FS_FORM") {
+                };
+            } else if (fundingRequest.funding_request_type == "FS_FORM") {
                 fundingDocument = {
                     "fundingType": fundingRequest.funding_request_type.toString(),
                     "gtcNumber": fundingRequest.fs_form.gt_c_number.toString(),
                     "orderNumber": fundingRequest.fs_form.order_number.toString()
-                }
+                };
```

Comment:
This is an example of why I don't like the internal snow linter. 

Inconsistent application of semicolons for if / else statements.